visionppi/.idea/codeStyles/Project.xml

```diff
@@ -3,6 +3,115 @@
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <codeStyleSettings language="XML">
```

Comment:
Please revert this change. Changes to any file in the .idea folder should never be pushed.

Reply:
Sorry I forgot to git ignore, correcting it